docs/source/python_api/mlflow.models.rst

```diff
@@ -6,3 +6,7 @@ mlflow.models
     :undoc-members:
     :show-inheritance:
 
+.. autoclass:: mlflow.models.model.ModelInfo
+    :members:
+    :undoc-members:
+    :exclude-members: ModelSignature
```

Comment:
This line removes the duplicate entry of `ModelSignature` (should not appear in `ModelInfo`) imported by https://github.com/mlflow/mlflow/blob/00bf853a4d832e168c3e7ff4f28e7b48bccd7ccf/mlflow/models/model.py#L36, which is to avoid a global import since pandas is not a dependency of mlflow-skinny.